src/lucky_web/date_helpers.cr

```diff
@@ -0,0 +1,47 @@
+module LuckyWeb::DateHelpers
```

Comment:
Can we call this `TimeHelpers` instead? Also this should be in `lucky_web` and under `Lucky` namespace now. Sorry for the hassle! This was recently changed. But better now than later

Reply:
Sure. you mean under `lucky_web` directory but call module `Lucky`. How about spec? I just changed the directory too.

Reply:
Looks like you got it right. `Lucky` namespace in `src/lucky` directory 👍

Reply:
It was `src/lucky` but you said `lucky_web` so I changed it again 😭

Reply:
Oops! I'll update my comment :P